### ISSUE TITLE
Discovery: Determine content types for pages with heading violations

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11135]
## Description
There are a number of pages on VA.gov that have accessibility heading violations, meaning that the heading structure does not follow logical order and some levels are skipped. One of the initial ideas to help editors avoid creating these errors when creating and updating content is to harden specific content types. This would templatize the pages and headings minimizing the editors ability to add free-form rich text editor content.

The most recent export of these violations was done in April 2022. The goal would be to parse that export and compare the pages with heading violations with the original VA.gov IA to recommend which content types might be most beneficial to harden. Also, look to see if there are patterns within the pages for content types that may not have been proposed in the original IA but are now needed (i.e. Campus Map).

## Additional Content

- The export can be found [here](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/9571#issuecomment-1164501471), along with original context
- Original VA.gov [IA for Facility sites](https://app.moqups.com/Rnc4BDEKrA/view/page/a9e1a59e9)

## Acceptance Criteria
- [x] Parse through heading violation export
- [x] Compare pages in export to original VA.gov IA
- [x] Determine which product each page belongs to (i.e. VAMC, Events, Vet Center, etc.)
- [x] Determine recommendations for hardened content types 
- [x] Create presentation readout

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidconlon** [comment]
Please add an AC to also delineate by Product (e.g. VAMC Product, Events product, etc) I think some of the old report had some events listed in it that is actually the events product rather than the VAMC Product because it's shared.

**Comment by laflannery** [comment]
AC added

**Comment by laflannery** [comment]
[Readout](https://docs.google.com/presentation/d/1XOto3GsUWLR1J_8clpypmUHYzpmDRNkv-wUjrjYj1-M/edit#slide=id.g16c2d2e3677_0_23)
